Engine/Spells/CastSpellInfo.cpp

```diff
@@ -3772,7 +3772,7 @@ void CastSpellInfoHelpers::CastSpell() {
             pPlayer->PlaySound(SPEECH_CastSpell, 0);
         }
         if (spell_sound_flag) {
-            pAudioPlayer->PlaySpellSound(pCastSpell->uSpellID, 0);
+            pAudioPlayer->PlaySpellSound(pCastSpell->uSpellID, -1);
```

Comment:
Let's use PID_INVALID instead of -1 here. Also would be cool to add docs to PlaySpellSound and spell out the semantics of this parameter.